.babelrc

```diff
@@ -1,3 +1,4 @@
 {
-  "presets": ["babel-preset-expo"]
+  "presets": ["babel-preset-expo"],
+  "ignore": "main.js"
```

Comment:
@mehdisadeghi this got slipped here but it is actually were the magic happens. This tells the babel js transpiler to avoid the `main.js` file since it was already compiled by the Clojurescript compiler.

We need this line to avoid having expo build times of over 20 minutes (see travis ci build history).

Reply:
Yeah makes sense!